python/llm/setup.py

```diff
@@ -272,16 +272,27 @@ def setup_package():
     all_requires += CONVERT_DEP
 
     # Linux install with -f https://developer.intel.com/ipex-whl-stable-xpu
-    xpu_requires = copy.deepcopy(all_requires)
-    xpu_requires.remove('torch')
-    xpu_requires += ["torch==2.0.1a0;platform_system=='Linux'",
-                     "torch==2.0.0a0+gitc6a572f;platform_system=='Windows'",
-                     "torchvision==0.15.2a0;platform_system=='Linux'",
-                     "intel_extension_for_pytorch==2.0.110+xpu;platform_system=='Linux'",
-                     "intel_extension_for_pytorch==2.0.110+gitba7f6c1;platform_system=='Windows'",
-                     "bigdl-core-xe==" + VERSION + ";(platform_system=='Linux' or platform_system=='Windows')",
-                     "bigdl-core-xe-esimd==" + VERSION + ";platform_system=='Linux'"]
-
+    xpu_20_requires = copy.deepcopy(all_requires)
+    xpu_20_requires.remove('torch')
+    xpu_20_requires += ["torch==2.0.1a0;platform_system=='Linux'",
```

Comment:
Besides, could you also remove the win supports for 20 as I stopped the internal release for win bigdl-core-xe from tonight. :)

Reply:
done